integrations/build_system/xmake.rst

```diff
@@ -0,0 +1,69 @@
+.. xmake:
+
+XMake
+======
+
+
+Install third-party packages:
+-----------------------------
+
+After version 2.2.5, xmake supports support for dependency libraries in third-party package managers.
+
+.. code-block:: lua
+   :caption: xmake.lua
+    
+    -- xmake.lua
+    
+    add_requires("conan::zlib/1.2.11@conan/stable", {alias = "zlib", debug = true})
+    add_requires("conan::OpenSSL/1.0.2s@conan/stable", {alias = "openssl",
```

Comment:
This keeps using the old ``OpenSSL/1.0.2s@conan/stable`` package. It should better use the ``openssl/1.1.1g`` (without user/channel), because those are the modern, mantained packages.

Reply:
Fixed! Sorry to keep you waiting.

Anything else missing?